ec/provider_config.go

```diff
@@ -18,7 +18,9 @@
 package ec
 
 import (
+	"context"
 	"fmt"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
```

Comment:
This import should be moved to line 28, and preserve the space between the stdlib imports and the external ones. It's something that we should have in our linter. I'll add that linter check so this doesn't happen again.

Reply:
Thanks @marclop for the review, have corrected the order of imports.